velox/core/ITypedExpr.h

```diff
@@ -23,12 +23,13 @@ namespace facebook::velox::core {
 /* a strongly-typed expression, such as literal, function call, etc... */
 class ITypedExpr {
  public:
-  ITypedExpr(std::shared_ptr<const Type> type) : type_{move(type)}, inputs_{} {}
+  ITypedExpr(std::shared_ptr<const Type> type)
```

Comment:
The internal linter asks to add "explicit" to this constructor:

```
Single-argument constructor missing explicit specifier.
A single-argument constructor for ITypedExpr is missing the explicit specifier.
A single-argument constructor may inadvertently be used as a type conversion constructor. Prefix the constructor with the 'explicit' keyword to avoid this.
If this is intentional, consider adding '/* implicit */' at the start of the constructor declaration to silence this lint warning.
```

Reply:
Now I wonder how did this test before? I just replace move with std::move.

Reply:
Just to clarify a linter warning is not the same as test failure.